app/src/main/java/org/thoughtcrime/securesms/database/model/NotificationMmsMessageRecord.java

```diff
@@ -64,50 +55,26 @@ public NotificationMmsMessageRecord(long id, Recipient conversationRecipient,
     this.transactionId   = transactionId;
   }
 
-  public byte[] getTransactionId() {
-    return transactionId;
-  }
-  public int getStatus() {
-    return this.status;
-  }
-  public byte[] getContentLocation() {
-    return contentLocation;
-  }
-  public long getMessageSize() {
-    return (messageSize + 1023) / 1024;
-  }
-  public long getExpiration() {
-    return expiry * 1000;
-  }
+  public byte[] getTransactionId() { return transactionId; }
 
-  @Override
-  public boolean isOutgoing() {
-    return false;
-  }
+  public int getStatus() { return this.status; }
+
+  public byte[] getContentLocation() { return contentLocation; }
+
+  public long getMessageSize() { return (messageSize + 1023) / 1024; }
+
+  public long getExpiration() {  return expiry * 1000; }
```

Comment:
Why reformat java? just convert to kotlin

Reply:
You make a convincing argument - done! :)